projects/Amlogic/packages/opengl-meson/system.d/libmali.service

```
@@ -0,0 +1,10 @@
+[Unit]
+Description=Choose libMali for the device
+Before=kodi.service
+
+[Service]
+Type=oneshot
+ExecStart=-/usr/bin/sh -c '[ ! -e /sys/devices/system/cpu/cpu4 ] && mount -o bind /usr/lib/libMali.so /usr/lib/libEGL.so && mount -o bind /usr/lib/libMali.so /usr/lib/libGLESv1_CM.so && mount -o bind /usr/lib/libMali.so /usr/lib/libGLESv2.so'
```

```suggestion
ExecStart=-/usr/bin/sh -c '[ ! -e /proc/device-tree/t82x@d00c0000/compatible ] && mount -o bind /usr/lib/libMali.so /usr/lib/libEGL.so && mount -o bind /usr/lib/libMali.so /usr/lib/libGLESv1_CM.so && mount -o bind /usr/lib/libMali.so /usr/lib/libGLESv2.so'
```

or maybe look at @vpeter4 / @Raybuntu suggestion? and move to a helper script rather than a oneliner in a service?